app/helpers/barcode_labels_helper.rb

```diff
@@ -24,7 +24,7 @@ def barcode_printing_form(
   end
 
   def printers_of_type(printer_types)
-    @printers.select { |printer| printer_types.include?(printer.type.name) } # rubocop:todo Rails/HelperInstanceVariable
+    @printers.select { |printer| printer_types.include?(printer.barcode_type) } # rubocop:todo Rails/HelperInstanceVariable
```

Comment:
checking that isn't meant to be barcode_type**.name**

Reply:
Thanks for checking, but no it's OK.  The name is implicitly what the barcode_type refers to in the API.

![image](https://github.com/user-attachments/assets/512b745d-fd5e-4bcf-8319-a2e846b2e5af)